README.md

```diff
@@ -8,23 +8,26 @@
 [travis-badge]: https://travis-ci.org/picostyle/picostyle.svg
 [travis]: https://travis-ci.org/picostyle/picostyle
 
-<div align="center">
-  <a href="https://github.com/morishitter/picostyle">
-    <img width="360px" src="http://morishitter.github.io/picostyle.svg">
-  </a>
-</div>
-<br>
+<a href="https://github.com/morishitter/picostyle">
```

Comment:
@lukejacksonn Maybe remove the logo from the README? It's already visible on the org page.

Reply:
I agree. Have removed 👍